spec/integration/interactive_mode_spec.rb

```diff
@@ -45,9 +45,10 @@
       owners: ['@mpospelov'],
       file_tree: { 'lib/new_file.rb' => 'bar' }
     ) do
-      expect_to_ask(<<~QUESTION)
+      expect_to_ask(<<~QUESTION, limited_to: %w[y r i q])
         Unknown owner: @foobar for pattern: lib/new_file.rb. Add owner to the OWNERS file?
         (y) yes
+        (r) rename owner
         (i) ignore owner in this session
         (q) quit and save
       QUESTION
```

Comment:
I think we can add an integration test for renaming owner flow too, something like
<details><summary>inside spec/support/integration_test_helper.rb</summary>

```
def expect_codeowners_to_eq(expected)
  expect(File.read(File.join(PROJECT_PATH, '.gihub', 'CODEOWNERS'))).to eq(expected.join("\n"))
end
```

</details>

<details><summary>inside spec/integration/interactive_mode_spec.rb</summary>

```
it 'runs with invalid_owner issue, rename flow' do
  start(
    codeowners: ['lib/new_file.rb @foobar'],
    owners: ['@mpospelov'],
    file_tree: { 'lib/new_file.rb' => 'bar' }
  ) do
    expect_to_ask(<<~QUESTION, limited_to: %w[y r i q]) { 'r' }
      Unknown owner: @foobar for pattern: lib/new_file.rb. Add owner to the OWNERS file?
      (y) yes
      (r) rename owner
      (i) ignore owner in this session
      (q) quit and save
    QUESTION

    expect_to_ask(<<~QUESTION, limited_to: %w[y r i q]) { '@mpospelov' }
      New owner: 
    QUESTION
    expect_codeowners_to_eq(['lib/new_file.rb @foobar'])
  end
end
```
</details>

Reply:
done